src/Fogger/Data/ChunkProducer.php

```diff
@@ -36,8 +36,10 @@ private function queueTableChunks(Table $table)
 
             return;
         }
-
-        $result = $this->sourceQuery->getAllKeysQuery($table)->execute();
+        
+        $query = $this->sourceQuery->getAllKeysQuery($table);
+        $query = $query->orderBy($table->getSortBy());
```

Comment:
Could probably chain these methods if you wanted but would need to double check. Fine this way too though.